cities.json

```diff
@@ -2104,186 +4732,213 @@
+  {"id": "2520407","name": "منطقة كاردهو","state_id": "25204"},
+
+  {"id": "2520501","name": "منطقة بيدوا","state_id": "25205"},
+  {"id": "2520502","name": "منطقة برهاكابا","state_id": "25205"},
+  {"id": "2520503","name": "منطقة دينسور","state_id": "25205"},
+  {"id": "2520504","name": "منطقة قساهدير","state_id": "25205"},
+
+  {"id": "2520601","name": "منطقة طوس مريب","state_id": "25206"},
+  {"id": "2520602","name": "منطقة ادادو","state_id": "25206"},
+  {"id": "2520603","name": "منطقة جوري عيل","state_id": "25206"},
+  {"id": "2520604","name": "منطقة أبودواق","state_id": "25206"},
+  {"id": "2520605","name": "منطقة بالانبل","state_id": "25206"},
+  {"id": "2520606","name": "منطقة غالهاريري","state_id": "25206"},
+  {"id": "2520607","name": "منطقة جودينلابي","state_id": "25206"},
+  {"id": "2520608","name": "منطقة الضهر","state_id": "25206"},
+  {"id": "2520609","name": "منطقة البور","state_id": "25206"},
+
+  {"id": "2520701","name": "بارديرا","state_id": "25207"},
+  {"id": "2520702","name": "بيليد هاو","state_id": "25207"},
+  {"id": "2520703","name": "الواك","state_id": "25207"},
+  {"id": "2520704","name": "دولو","state_id": "25207"},
+  {"id": "2520705","name": "غارباهارى","state_id": "25207"},
+  {"id": "2520706","name": "لوك","state_id": "25207"},
+  {"id": "2520707","name": "بوردو","state_id": "25207"},
+
+  {"id": "2520801","name": "بلدوين","state_id": "25208"},
+  {"id": "2520802","name": "جلالقسي","state_id": "25208"},
+  {"id": "2520803","name": "بولوبوردي","state_id": "25208"},
+  {"id": "2520804","name": "موكوكوري","state_id": "25208"},
+  {"id": "2520805","name": "هالجان","state_id": "25208"},
+  {"id": "2520806","name": "بوك اكابل","state_id": "25208"},
+  {"id": "2520807","name": "ماتابان","state_id": "25208"},
+  {"id": "2520808","name": "ماهاس","state_id": "25208"},
+
+  {"id": "2520901","name": "بوالي","state_id": "25209"},
+  {"id": "2520902","name": "جليب","state_id": "25209"},
+  {"id": "2520903","name": "ساكو","state_id": "25209"},
+  {"id": "2520904","name": "سالاجل","state_id": "25209"},
+
+  {"id": "2521001","name": "كسمايو","state_id": "25210"},
+  {"id": "2521002","name": "جمامه","state_id": "25210"},
+  {"id": "2521003","name": "أفمدو","state_id": "25210"},
+  {"id": "2521004","name": "بطاطي","state_id": "25210"},
+
+  {"id": "2521101", "name": "گالكايو", "state_id": "25211"},
+  {"id": "2521102", "name": "گالدوگوب", "state_id": "25211"},
+  {"id": "2521103", "name": "جاريبان", "state_id": "25211"},
+  {"id": "2521104", "name": "هوبيو", "state_id": "25211"},
+
+  {"id": "2521201", "name": "غاروي", "state_id": "25212"},
+  {"id": "2521202", "name": "إيل", "state_id": "25212"},
+  {"id": "2521203", "name": " بيرتينل", "state_id": "25212"},
+  {"id": "2521204", "name": " دانغورايو.", "state_id": "25212"},
+
+  {"id": "2521301", "name": "عيرجابو", "state_id": "25213"},
+  {"id": "2521302", "name": "دهار", "state_id": "25213"},
+  {"id": "2521303", "name": "بادهان", "state_id": "25213"},
```

Comment:
Snaag is a bit of issue, different sources can't agree on a single district's distribution.
Check:
https://www.mappr.co/political-maps/somalia-map/ 
https://www.citypopulation.de/en/somalia/admin/15__sanaag/
https://en.wikipedia.org/wiki/Sanaag#Districts
Let me know your take in this.

Reply:
Somalia in general really difficult to review since there is disputes on dividing the main land, for the cas of snaag we can assume it is part of somalialand so you can use this reference here https://en.wikipedia.org/wiki/Sanaag#Districts